cxx-qt-gen/src/extract.rs

```diff
@@ -960,16 +951,20 @@ pub fn extract_qobject(
                                     _others => original_trait_impls.push(original_impl.to_owned()),
                                 }
                             } else {
-                                let mut extracted = extract_invokables(
-                                    &original_impl.items,
-                                    cpp_namespace_prefix,
-                                    &qt_ident,
-                                )?;
-
-                                object_invokables.append(&mut extracted.invokables);
-                                object_normal_methods.append(&mut extracted.normal_methods);
+                                // Invokables aren't on the impl T block
+                                object_normal_methods.append(&mut original_impl.items);
```

Comment:
Should using `#[invokable]` outside of `impl Qt<RustObj>` cause an error?

Reply:
I think it fails to build as there is no macro called `invokable` :-)